spec/systems/logout_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Logout', type: :system do
+  context 'given logout click action' do
```

Comment:
I think we no need this context as we know the user can logout by click to the logout button :)

Reply:
Fixed in 6067c4c51e142495d76a0477027568a1ab33be9f